lib/bundles/inspec-compliance/api/login.rb

```diff
@@ -0,0 +1,145 @@
+# encoding: utf-8
+# author: Christoph Hartmann
+# author: Dominik Ricter
+# author: Jerry Aldrich
+
+module Compliance
+  class API
+    module Login
+      def login(options)
+        options['server_type'] = Compliance::API.determine_server_type(options['server'], options['insecure']).to_s
+
+        return Login::ComplianceServer.login(options) if options['server_type'] == 'compliance'
```

Comment:
Here is where I'd raise an exception if we don't know the server type since this is actually where an error should be raised to the user:

```ruby
if options['server_type'] == :compliance
  Login::ComplianceServer.login(options)
elsif options['server_type'] == :automate
  Login::AutomateServer.login(options)
else
  raise CannotDetermineServerType, # .....
end
```